templates/overlappingWithModule.html

```diff
@@ -0,0 +1,37 @@
+$def with (moduleCode, overlappingList)
```

Comment:
![image](https://cloud.githubusercontent.com/assets/12386860/24054623/d5318b70-0b77-11e7-963a-538d9e3ce386.png)

1. **Bold the title and position it to center**
2. **Add description of what the page shows**
3. URL contains the AY-Sem, not sure if the table still requires a column for AY-Sem.
@helloqx @nlzz22 @xaterz opinions?

4. @a0129998 will this be showing top-10 overlapping modules w.r.t. the target module? If so, need to ensure page description reflects this, and the search + pagination can be disabled.